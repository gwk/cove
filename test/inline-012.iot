{
  'cmd': '$PROJ/cove.py test/inline.py 0 12',
}
