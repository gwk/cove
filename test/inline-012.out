cove report:

__main__: test/inline-{}.py: 28 lines; [37m11 trivial[0m; 17 traceable; 17 covered; 0 relaxed; 0 ignored; 0 ignored but covered; 0 not covered.
